on-boarding/task-2/main.dart

```diff
@@ -22,8 +22,14 @@ void main() {
 
     switch (choice) {
       case 1:
-        print('Enter task title:');
-        String title = stdin.readLineSync() ?? '';
+        String title = '';
```

Comment:
Code Reusability:
One improvement that can be made is to reduce code duplication. In the main() function, the code for handling input validation of the title is repeated in both the 'Edit task' and 'Delete task' cases. It's better to refactor this code into a separate function to promote code reusability and improve maintainability. For example, you could create a function called getTitleFromUser() that handles the input validation for the title.